cmd/kubeadm/app/master/apiclient.go

```diff
@@ -219,3 +222,41 @@ func SetMasterNodeAffinity(meta *api.ObjectMeta) {
 	}
 	meta.Annotations[api.AffinityAnnotationKey] = string(affinityAnnotation)
 }
+
+func createDummyDeployment(client *clientset.Clientset) {
+	fmt.Println("<master/apiclient> attempting a test deployment")
+	dummyDeployment := NewDeployment("dummy", 1, api.PodSpec{
+		SecurityContext: &api.PodSecurityContext{HostNetwork: true},
+		Containers: []api.Container{{
+			Name:  "dummy",
+			Image: images.GetAddonImage("pause"),
+		}},
+	})
+
+	wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
```

Comment:
This seems like it could run forever silently. Can we add a timeout or occasionally report statuses?